src/tap/test/boundary_macros.ok

```diff
@@ -12,8 +12,7 @@
 [INFO ODB-0130]     Created 54 pins.
 [INFO ODB-0131]     Created 5 components and 160 component-terminals.
 [INFO ODB-0303] The initial 57 rows (24054 sites) were cut with 5 shapes for a total of 13 rows (577 sites).
-[INFO TAP-0004] Inserted 26 endcaps.
-[INFO TAP-0006] Inserted 89 top/bottom cells.
-[INFO TAP-0007] Inserted 11 cells near blockages.
+[INFO TAP-0002] Added 8 corner cells
+[INFO TAP-0003] Added 196 boundary cells
```

Comment:
Why the large shift in # of cells?

Reply:
This testcase doesn't work in the original code. It only places some of the cells needed, now it's placing all the end caps on the perimeter. This increase is expected. #2300